convene-web/app/views/guides/_rooms.md

```diff
@@ -0,0 +1,132 @@
+  originals and prints for sale);
+- an office for project materials and updates; or
+- a front porch where you broadcast your stream-of-consciousness thoughts to the
+  world and catch up with your neighbors and acquaintances.
+
+To tune each [Room] for its purpose, fill it with [Furniture] that facilitates
+activities or collect and share information.
+
+[Rooms] may be added by [Space Owners] in [Space Configuration].
+
+## Entering Rooms
+
+Rooms can be entered in one of two ways:
+
+1. Opening the [Room Link] _or_
+2. Browsing the [Room Directory]
+
+### Room Links
+
+Room Links are specific to a particular room, and are the easiest way to share
+what's in a Room with [Neighbors] and [Guests]. You can find the [Room Link]
+through the [Share Action].
+
+## Room Privacy and Security
+
+When creating a [Room], [Space Owners] may want to think about who should be
+allowed to see or enter the Room. There are three affordances for creating the
+ideal privacy and security for a particular [Room].
+
+1. [Hiding a Room](#hidden-rooms)
+1. [Private Rooms](#private-rooms)
+1. [Locking a Room](#locked-rooms)
+
+### Hidden Rooms
+
+While hidden rooms may only be found in the [Room Directory] by [Space Owners]
+and [Neighbors] or [Space Members] who have entered the Room before, **_anyone
+with the [Room Link] may access a hidden room!_**
+
+If you want to keep the activities or information in a room private, you will
+also want to make it [Locked](#locked-rooms) or [Private](#private-rooms).
+
+Hidden Rooms can be found in the [Room Directory] by [Space Owners]; as well as
+by [Neighbors] or [Space Members] who entered the Room since it was last Hidden.
+
+### Private Rooms
+
+Private Rooms can only be found and entered by [Space Members] and [Space
+Owners].
+
+If you want only [Space Members] who know the [Room Link] or have entered the
+Room before to be able to find the room, consider also making it
+[Hidden](#hidden-rooms).
+
+If you want to further restrict access to only specific [Space Members] and
+[Space Owners], consider [Locking the Room](#locked-rooms)
+
+### Locked Rooms
+
+Sometimes, a [Room] wants a bit of privacy. [Rooms] can be _Locked_ by any
+[Space Member]. When locked, [Rooms] require an _[Access Code]_ to enter. A
+[Room] can be locked and unlocked as part of [Room Configuration].
+
+#### How To Lock a Room
+
+1. Open the Room's configuration.
+2. Select "Locked" from the "Access Level" drop-down.
+3. Enter an [Access Code].
+4. Save your changes.
+
+[Locked Rooms] have a "Lock" icon on their door. People attempting to enter a
+[Locked Room] will be prompted to enter the Access Code.
+
+People who entered the [Locked Room] before may enter automatically, so long as
+the [Access Code] has not changed since they last entered.
+
+#### Access Codes
+
+Access Codes are a shared secret between everyone who can enter a locked room.
+They can be changed by the [Space Owners]. A [Room][access code] can be added,
```

Comment:
I think there needs to be a space between [Room] and [access code].

Reply:
Nice catch!